src/model/encoding/__tests__/convertFromHTMLToContentBlocks-test.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails isaac, oncall+ui_infra
+ */
+
+'use strict';
+
+jest.disableAutomock();
+
+const convertFromHTMLToContentBlocks = require('convertFromHTMLToContentBlocks');
+const DefaultDraftBlockRenderMap = require('DefaultDraftBlockRenderMap');
```

Comment:
should be requires starting with a capital letter, then a blank line, then requires starting with a lowercase letter